eth/chains/base.py

```diff
@@ -650,12 +672,15 @@ def apply_transaction(
         receipt, computation = vm.apply_transaction(base_block.header, transaction)
         header_with_receipt = vm.add_receipt_to_header(base_block.header, receipt)
 
-        # since we are building the block locally, we have to persist all the incremental state
+        # since we are building the block locally,
+        # we have to persist all the incremental state
         vm.state.persist()
-        new_header: BlockHeaderAPI = header_with_receipt.copy(state_root=vm.state.state_root)
+        new_header: BlockHeaderAPI = header_with_receipt.copy(
+            state_root=vm.state.state_root
+        )  # noqa: E501
```

Comment:
Scan file for unnecessary `# noqa: E501` uses

Reply:
deleted